common/lib/capa/capa/responsetypes.py

```diff
@@ -1931,6 +1937,22 @@ def get_score(self, student_answers):
                 if compare_with_tolerance(student_float, correct_float, expanded_tolerance):
                     is_correct = 'partially-correct'
 
+        # Compare with additional answers.
+        if is_correct == 'incorrect':
+            additional_answer_idx = 0
```

Comment:
Can we rename this to avoid confusion, maybe simple `index` or `temp_additional_answer_idx` ?